static/install/compose

```diff
@@ -32,7 +32,7 @@ services:
       - appwrite
 
   appwrite:
-    image: appwrite/appwrite:1.5.7
```

Comment:
the executor was updated too

Reply:
Yup, that's updated down below:

https://github.com/appwrite/website/blob/702fa9dba3eaa1ec22d436b7809caba9cf903514/static/install/compose#L749